docs/UserGuide.md

```diff
@@ -40,22 +47,125 @@ for more information:)
+
+![one entry in the student list](images/UIexplanation/personcard.png)
+
+Shown here is:
+
+| Description                                     | From picture above    |
+|-------------------------------------------------|-----------------------|
+| **Index** of the student in the student list    | Index of 2            |
+| **Name** of the student                         | Charlotte Oliverio    |
+| **Tags** given to the student                   | One tag, `neighbours` |
+| **Matriculation/Student number** of the student | A9321028C             |
+| **Module** that the student is taking           | CS2030S               |
+| **Group** that the student is under             | T02                   |
+| **Email** of the student                        | charlotte@example.com |
+
+Note that the group of the student can be set to anything you want, so the meaning of it is derived upon 
+your own usage. The example given, `T02`, is intended to imply that the student belongs to Tutorial Group 2
+
+<br>
 
-   * **`add`**` n/John Doe a/A0123456P e/johnd@example.com m/CS2103T g/W12` : Adds a student
+#### Task List
 
-     named `John Doe` to the Contact List.
+This is where you can see the tasks you have created.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+![task list](images/UIexplanation/tasklist.png)
 
-   * **`clear`** : Deletes all contacts.
+Taking a closer look at one of them:
 
-   * **`exit`** : Exits the app.
+![one entry in the task list](images/UIexplanation/taskcard.png)
 
-7. Refer to the [Features](#features) below for details of each command.
+Shown here is:
+
+| Description                                     | From picture above    |
+|-------------------------------------------------|-----------------------|
+| **Index** of the task in the task list          | Index of 1            |
+| **Description** of the task                     | interview for project |
+| **Deadline** of the task                        | 2022-02-25, 12:00     |
+
+<br>
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Typical workflow
+[<sub><sup>Back to top</sup></sub>](#table-of-contents)
+
+The typical workflow for a Teaching Assistant is provided as follows:
+
+1. Launch the application.
+
+
+2. Add students to the student list via the [add command](#adding-a-student-add) or the [import command](#importing-a-csv-file-to-automatically-add-students-import-csv).
+    1. Edit students if necessary via the [edit command](#editing-a-student-edit).
+
+
+
+3. Perform any of the actions below whenever necessary:
+    1. Find students via the [find command](#locating-students-find).
+    2. Mail students via the [various](#mailing-a-particular-student-based-on-index-mail-index).
+       [mail](#mailing-student-subset-based-on-arguments-mail-x) [commands](#bulk-emails-to-everyone-on-the-database-mail-all).
```

Comment:
We don't have a section/subheader for mailing commands. Would you want to create one then?

Reply:
Oh! I thought we had implemented the sectioning that we had discussed in our group chat a while back! Sorry, that's an oversight at my end too- but yes, most definitely! Having a "mailing commands" section can be a nice categorization that will also allow for more frequent chances to go back to the toc:) If you can implement this section and link point 2 to that header, then this will be great in my opinion